app/actions/websocket/index.ts

```diff
@@ -298,6 +298,7 @@ export async function handleEvent(serverUrl: string, msg: WebSocketMessage) {
             break;
 
         case WebsocketEvents.DIRECT_ADDED:
+        case WebsocketEvents.GROUP_ADDED:
```

Comment:
This event is for when you are added to a group message, nothing to do with custom groups.